CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-All patches are welcome. If you're ading a new example, please do create an issue before submitting a patch.
+All patches are welcome. If you're adding a new example, please do create an issue before submitting a patch.
 dcd
```

Comment:
Hey @alessandrocuzzocrea , Thanks for the patch.
 
Notice this "dcd" in L3, could you please also fix that? (Damn, I might be sleeping when I wrote this CONTRIBUTING.md)